cmd/dep/init.go

```diff
@@ -113,39 +119,26 @@ func (cmd *initCommand) Run(ctx *dep.Ctx, args []string) error {
 	sm.UseDefaultSignalHandling()
 	defer sm.Release()
 
-	pd, err := getProjectData(ctx, pkgT, cpr, sm)
-	if err != nil {
-		return err
-	}
-	m := &dep.Manifest{
-		Dependencies: pd.constraints,
-	}
-
-	// Make an initial lock from what knowledge we've collected about the
-	// versions on disk
-	l := &dep.Lock{
-		P: make([]gps.LockedProject, 0, len(pd.ondisk)),
+	var rootAnalyzer rootProjectAnalyzer
+	var analyzer gps.ProjectAnalyzer
+	if cmd.skipTools {
+		rootAnalyzer = newGopathAnalyzer(ctx.Loggers, ctx, pkgT, cpr, sm)
+		analyzer = dep.Analyzer{}
+	} else {
+		rootAnalyzer = compositeAnalyzer{
+			Analyzers: []rootProjectAnalyzer{
+				newGopathAnalyzer(ctx.Loggers, ctx, pkgT, cpr, sm),
+				newImportAnalyzer(ctx.Loggers, sm),
+			}}
```

Comment:
Just trying to understand, the `compositeAnalyzer` would also be added to `analyzer.go`'s `DeriveManifestAndLock()` so that the dependency projects are also analyzed for dep, glide, godep, etc config in them, right?